### DISCUSSION TITLE
about/index

### DISCUSSION THREAD

**Discussion by giscus** [Jared-ZDC/markel#22] [General]
# about/index

闲看庭前花开花落，坐等天际云卷云舒 关键描述：  湖大本科毕业 IT从业狗 软硬件性能优化 Linux爱好者 代码阅读专家 芯片验证行业多年 AI起步学者 量化交易入门狗 幻想财富自由 大厂旅行者 不想做股票投资的程序员不是好管理者 交流邮箱: &#109;&#x61;&#114;&#107;&#x65;&#108;&#x5f;&#x7a;&#x68;&#117;&#64;&#x31;&#50;&amp;

https://markel.top/about/index.html

**Comment by Jared-ZDC** [comment]
评论不了？

**Reply by Jared-ZDC** [reply]
try again, it's OK~~
